File: pruebas.js

const copyToClipboard = str => {
    if (navigator && navigator.clipboard && navigator.clipboard.writeText)
      return navigator.clipboard.writeText(str);
    return Promise.reject("The Clipboard API is not available.");
};

document.getElementById('export')
document.getElementById('export')
{"type":"FeatureCollection","features":[{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[-76.518016,3.43216],[-76.529775,3.443641],[-76.534195,3.432503],[-76.518016,3.43216]]]}}]}


var aux = document.getElementById('export').innerHTML;
aux.select();
document.execCommand('copy');
alert("copiadoo!!")

File: pruebas.py
import json

if __name__ == "__main__":
    coords_dict = json.loads("""{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[-76.51896,3.442056],[-76.515098,3.434174],[-76.528187,3.432546],[-76.51896,3.442056]]]}}""")
    coords= coords_dict['geometry']['coordinates'][0]
    print(coords)


